### PR TITLE
Css readme

### PR DESCRIPTION
**Pull request by vmipfdr**
- Related Issue (include '#'):  #434 (and the rest are actually closed)

- Description of changes made:  Wrote a readme for CSS Variables and BEM, and while finding a reference for the readme, fixed one variable.  

- Is the feature complete/bug resolved/etc..:  Yes

- Any known bugs/strange behavior:  No

- Is there specific feedback you would like on these changes:  Nope

- Screenshot(s):  N/A

### CODE REVIEW

**Comment by vmipfdr** [comment]
@ZakkMan Do not do anything with the js files.  I couldn't figure out how to NOT push them with the readme even though I specifically only added the readme file and committed that today.

File: client/src/components/Banner/Banner.css [hunk 1]
```diff
@@ -14,6 +14,6 @@
   text-align: center;
   font-family: var(--default-font);
   font-size: 48px;
-  color: #fafafa;
+  color: var(--white);
   line-height: 72px;
 }
```
Comment (ZakkMan) [inline client/src/components/Banner/Banner.css:19]: Can you remove these changes that are not part of the documentation?

File: client/src/CSS_Readme.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+# Description
```
Comment (ZakkMan) [inline client/src/CSS_Readme.md:1]: This all looks good. Could it maybe go in the existing [client-side README](https://github.com/CotripperPlatform/CoTrip/tree/develop/client)?

**Comment by vmipfdr** [comment]
Closing this pull request because we put the readme in the wiki, so we are removing this from the code base.  No content lost, just adjusting location of content.

**Pull request closed without merging**